user-guide/live-traffic/auto-responder/rules-edtor.md

```diff
@@ -0,0 +1,115 @@
+
+- __EXACT:URL__ - Will match URL that is identical to the passed **URL** parameter. Note that the __EXACT:__ rule is case-sensitive.
+```
+EXACT:www.example.com/Path1/query=foo
+
+www.example.com/Path1/query=foo // (MATCH)
+www.example.com/PATH1/query=foo // (No Match - mismatched case)
+```
+
+### Regular Expressions
+
+Fiddler Everywhere supports regular expressions syntax for expressions that begin with __regex:__. The regular expression will be used to replace the inbound URL with the string in the Actions row. Use __.+__ to match a sequence of one or more characters or __.*__ to match zero or more characters. Use __^__ at the front of your regex to mean "Start of the URL" and use __$__ at the tail of the regex to mean "End of the URL".
+
+- __regex:.+__ - Will match all URLs
+```
+www.example.com/Path1/query=foo // (MATCH)
+```
+
+- __regex:.+.jpg$__ - Will match URLs that are ending with __.jpg__
+```
+www.example.com/Path1/query=foo.jpg&bar // (MATCH)
+www.example.com/Path1/query=example.jpg // (MATCH)
+```
+
+- __regex:.+.(jpg|gif|bmp)$__ - Will match URLs that are ending with the listed image extensions (case-sensitive!).
+```HTML
+www.example.com/Path1/query=foo.jpg&bar // (No Match - imnproper ending)
+www.example.com/Path1/query=exam ple.jpg // (MATCH)
+www.example.com/Path1/query=foo.JPG // (No Match - mismatched case)
+www.example.com/Path1/query=somegif.gif // (MATCH)
+```
+
+## Action Strings
+
+Enter the __action string__. Beyond simply returning files, the __Auto Responder__ can perform some specific actions:
+
+- __filename__  - Return contents of the filename as the response.
+- __http://targetURL__ - Returns the content of the __targetURL__ as the response.
+- __*redir:http://targetURL__ - Returns a HTTP Redirect to the target URL. Unlike the simple URL rule, this ensures that the client knows where its request is going so proper cookies are sent, etc.
+- __*bpu__ - Breaks on request before hitting the server. Non-final acti0on.
+- __*delay:####__ - Delay sending request to the server by __####__ of miliseconds. Non-final action.
+- __*header:Name=Value__ - Set the Request header with the given __Name__ to the specified __Value__. If no header of that name exists, a new header will be created. Non-final action.
+- __*flag:Name=Value__ - Set the Session Flag, with the given __Name__ to the specified __Value__. If no header of that name exists, a new header will be created. Non-final action.
+- __*CORSPreflightAllow__ - Returns a response that indicates that CORS is allowed.
+- __*reset__ - Reset the client connection immediately using a TCP/IP RST to the client.
+- __*drop__ - Close the client connection immediately without sending a response.
+- __*exit__ - Stop processing rules at this point.
+
+Rules with Non-final actions will allow the request to match multiple Auto Responder rules. As soon a rule specifying a final action is reached, the matching process exists, and no further rules are processed for that session.
```

Comment:
Perhaps it is a good idea to mark the "Final actions" above as well, just to make it consistent. You can also consider turning this into a table, I think it will look good - Action String | Description | Action Type